crates/sui-cluster-test/src/main.rs

```diff
@@ -405,13 +405,9 @@ impl ClusterTest {
 
 #[tokio::main]
 async fn main() {
-    let config = telemetry_subscribers::TelemetryConfig {
-        service_name: "cluster-test".into(),
-        enable_tracing: std::env::var("SUI_TRACING_ENABLE").is_ok(),
-        json_log_output: std::env::var("SUI_JSON_SPAN_LOGS").is_ok(),
-        ..Default::default()
-    };
-    let _guard = telemetry_subscribers::init(config);
+    let _guard = telemetry_subscribers::TelemetryConfig::new(env!("CARGO_BIN_NAME"))
```

Comment:
Sorry can you remind me what `env!()` does again?  Also `.with_env()`?

Reply:
[env!](https://doc.rust-lang.org/stable/std/macro.env.html)

`.with_env()` is a new function i added to the TelemetryConfig to populate things based on the existing environment.